packages/db/scripts/cmds/db-sql.js

```diff
@@ -15,9 +15,8 @@ const { Client } = pg
  * @param {Object} [opts]
  * @param {boolean} [opts.reset]
  * @param {boolean} [opts.cargo]
- * @param {boolean} [opts.testing]
  */
-export async function dbSqlCmd ({ reset, cargo, testing } = {}) {
```

Comment:
We should keep the ability to setup non-testing version for running locally.

Reply:
so, use the flag testing instead of cargo.